README.md

```diff
@@ -64,10 +64,10 @@ export function getUsersModule(): IModule<IUserState> {
 * Create a `ModuleStore`
 
 ```typescript
-import {configureStore, IModuleStore} from "redux-dynamic-modules";
+import {createStore, IModuleStore} from "redux-dynamic-modules";

```

Comment:
Shouldn't this be from redux now?

Reply:
And in all of the examples

Reply:
createStore is a helper that we wrote for quickly using Module store with enhancer, so the enhancer is not exposed when not needed.